tests/schema.py

```diff
@@ -307,7 +307,7 @@ class DecimalPrimaryKey(dj.Lookup):
     definition = """
     id  :  decimal(4,3)
     """
-    contents = zip((0.1, 0.25, 3.99))
+    contents = list(zip((0.1, 0.25, 3.99)))
```

Comment:
Strangely, the logic in `test_fetch::test_offset` would exhaust this zip object, causing it to be empty when `test_fetch::test_decimal` was run, causing the latter test to fail.

It's possible that `ORDER BY` on the upstream table causes `DecimalPrimaryKey` to also sort (and therefore exhaust the zip object):

https://github.com/ethho/datajoint-python/blob/761c1663f4a2be45e521e84f3d33ef79b34b3ba4/tests/test_fetch.py#L205-L207

```python
    def test_offset(self, schema_any, lang, languages):
        """Tests offset"""
        cur = lang.fetch(limit=4, offset=1, order_by=["language", "name DESC"])
```

Reply:
zip can only be used once. Once exhausted, it's empty. So if you are using the same code more than once to populate a lookup table, you need to convert into a list first.